exp-player/addon/components/exp-frame-base/component.js

```diff
@@ -139,7 +139,7 @@ export default Ember.Component.extend({
             this.send('setTimeEvent', 'nextFrame');
             this.send('save');
             this.sendAction('next');
-            window.scrollTo(0, 0);
+            //window.scrollTo(0, 0);
```

Comment:
Question: Can you please comment on why this was turned off?

`exp-frame-base` is used by all experiments. A quirk of ember is that by default, pages don't automatically scroll back to the top when you change the page (next, previous, route transitions, etc). `window.scrollTo(0,0)` is intended to give people a more normal page change experience, and turning it off in the base frame would affect experiments by other groups.

Perhaps I can help identify a good alternative?